README.md

```diff
@@ -1,3 +1,24 @@
-# NewXKitBot
+# New-XKit Bot
+[![Gitter](https://badges.gitter.im/Join%20Chat.svg)](https://gitter.im/new-xkit/NewXKitBot?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge&utm_content=badge)
 
-[![Gitter](https://badges.gitter.im/Join%20Chat.svg)](https://gitter.im/new-xkit/NewXKitBot?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge&utm_content=badge)
\ No newline at end of file
+### What is New-XKit Bot?
+New-XKit Bot is the team behind [New XKit](https://github.com/new-xkit/XKit)'s helping little assistant, written in JavaScript for [Node.js](https://nodejs.org/). It's here to make the lives of the New-XKit team just a tiny bit easier!
+
+### What does it do?
+For now, the bot has 1 primary feature, which is bridging the support IRC channel on freenode to the gitter support channel.
+More functionality is might be added in the future, depending on our needs.
+
+### How do I run it myself?
+1. Install [Node.js](https://nodejs.org/)
+2. Clone this repository into a folder
+3. Copy the ``default.js`` file in the ``config`` subfolder and rename it to ``local.js``
+4. Change whatever you want in the ``local.js`` and remove whatever values you left as default
+5. Run the bot with ``node main.js``
+
+### How do I contribute?
+1. Fork the repository
+2. Make your changes in the repository
+3. Open a PR with your changes
```

Comment:
For readers who don't know what a PR is: it's a pull request. I know we're hosting this on Github, but I don't think all Github visitors know what a PR is.